### PR TITLE
XS✔ ◾ kpis - make it clearer they are just a convo starter

### PR DESCRIPTION
**Pull request by bradystroud**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

feedback from @adamcogan 

> 2. What was changed?
also uploaded new image from @bettybondoc 
✏️

> 3. Did you do pair or mob programming (list names)?

✏️

### CODE REVIEW

File: rules/employee-kpis/rule.md [hunk 1]
```diff
@@ -49,18 +49,14 @@ The result is a final score and % showing the performance of the employee.
...
-::: warning
-KPIs are just a conversation starter.
-:::
-
 ### Metrics to measure
 
 There's lots of metrics that can be measured, and the right ones are going to depend on what is important to the organization.
 
-For example, SSW is a consulting company and these are the metrics for an consultant. [template KPIs](https://github.com/SSWConsulting/SSW.EmployeeKPIs)
+For example, SSW is a consulting company and these are the metrics for an consultant. [template KPIs.](https://github.com/SSWConsulting/SSW.EmployeeKPIs)
```
Comment (bettybondoc) [inline rules/employee-kpis/rule.md:60]: Change from 
an consultant
to 
**_a_** consultant

File: rules/employee-kpis/rule.md [hunk 2]
```diff
@@ -26,13 +26,13 @@ As the company grows and more people join, 2 problems start to occur:
 
 For these reasons, it's important for all the review managers to get aligned about what topics and performance indicators need to be covered, and to ensure scope is limited.
 
-Key Performance Indicators (KPIs) are a tool for standardizing how performance is measured at a company and ensuring employees are aware of what they need to focus on.
+Key Performance Indicators (KPIs) are a tool for standardizing how performance is measured at a company and ensuring employees are aware of what they need to focus on. However, everybody brings much more to their companies than just statistics, KPIs are just a conversation starter.
```
Comment (bettybondoc) [inline rules/employee-kpis/rule.md:29]: Change from
, KPIs are just a conversation starter.
to
**_;_** KPIs are just a conversation starter.

(just change the comma to a semi-colon)

**Review by bettybondoc — CHANGES_REQUESTED** [review]

File: rules/employee-kpis/rule.md (hunk 2 above)
Comment (bradystroud) [inline rules/employee-kpis/rule.md:29]:
```suggestion
Key Performance Indicators (KPIs) are a tool for standardizing how performance is measured at a company and ensuring employees are aware of what they need to focus on. However, everybody brings much more to their companies than just statistics; KPIs are just a conversation starter.
```

**Review by bettybondoc — APPROVED** [review]
LGTM!

File: rules/employee-kpis/rule.md [hunk 3]
```diff
@@ -49,18 +49,14 @@ The result is a final score and % showing the performance of the employee.
...
-::: warning
-KPIs are just a conversation starter.
-:::
-
 ### Metrics to measure
 
 There's lots of metrics that can be measured, and the right ones are going to depend on what is important to the organization.
 
-For example, SSW is a consulting company and these are the metrics for an consultant. [template KPIs](https://github.com/SSWConsulting/SSW.EmployeeKPIs)
+For example, SSW is a consulting company and these are the metrics for a consultant. [template KPIs.](https://github.com/SSWConsulting/SSW.EmployeeKPIs)
```
Comment (bettybondoc) [inline rules/employee-kpis/rule.md:60]: ``

**Pull request merged**